app/models/manageiq/providers/openshift/inventory/collector/container_manager.rb

```diff
@@ -0,0 +1,11 @@
+class ManageIQ::Providers::Openshift::Inventory::Collector::ContainerManager < ManageIQ::Providers::Kubernetes::Inventory::Collector::ContainerManager
+  def templates
+    @templates ||= openshift_connection.get_templates
+  end
+
+  private
+
+  def openshift_connection
+    @openshift_connection ||= manager.connect
```

Comment:
please specify explicit `connect(:service => "openshift")`.
I want to get rid of openshift manager defaulting to openshift, it voilates Liskov principle.

Reply:
Done